packages/babel-plugin-import-jsx-pragma/index.js

```diff
@@ -39,40 +42,65 @@ module.exports = function( babel ) {
 
 	return {
 		visitor: {
-			JSXElement( path, state ) {
+			JSX( path, state ) {
 				if ( state.hasUndeclaredScopeVariable ) {
 					return;
 				}
 
 				const { scopeVariable } = getOptions( state );
 				state.hasUndeclaredScopeVariable = ! path.scope.hasBinding( scopeVariable );
 			},
+			JSXFragment( path, state ) {
+				if ( state.hasUndeclaredScopeVariableFrag ) {
+					return;
+				}
+
+				const { scopeVariableFrag } = getOptions( state );
+				if ( scopeVariableFrag === null ) {
+					return;
+				}
+
+				if ( path.type === 'JSXFragment' ) {
```

Comment:
When would this ever not be true, given that the property name on which the function is run is named `JSXFragment` ?

Reply:
This was redundant, thanks. I've removed it.